### PR TITLE
Mothball the 0.5 docs

### PR DESCRIPTION
**Pull request by robdodson**
PTAL @ebidel @addyosmani @arthurevans @jeffposnick

Aims to fix https://github.com/Polymer/docs/issues/1301

### CODE REVIEW

File: robots.txt [hunk 1]
```diff
@@ -0,0 +1,2 @@
+User-agent: *
+Disallow: /0.5/*
```
Comment (ebidel) [inline robots.txt]: just `Disallow: /0.5/` to get the entire directory. http://www.robotstxt.org/robotstxt.html

**Comment by ebidel** [comment]
Thinking we should probably do this now that after seeing https://github.com/Polymer/docs/issues/1301. 

In the first link though, it should [automagically](https://github.com/Polymer/docs/blob/master/app.yaml#L105) get redirected to the latest (1.0) version of that unversioned doc. It's too bad we also changed the urls between 1.0 and 0.5. Users could have changed the URL and have things work :\

**Comment by arthurevans** [comment]
Is the goal here to stop giving people 0.5 results?

If we do this via robots.txt, I don't know how long it takes for stuff to get dropped from the indexes. It could be a long time, and it's not going to be reliable. (i.e., it may cause the 0.5 results to just rot slowly, rather than get removed cleanly in one go).

Through webmaster tools, we can _explicitly_ request that the 0.5 pages be removed from the index... but in either case, if we remove the results from Google's index, the on-site search will stop showing 0.5 results, too.

**Comment by ebidel** [comment]
Good point on the CSE not getting results either. I guess the question we need to answer is what @arthurevans said. Do we care about 0.5 results?

**Comment by robdodson** [comment]
Personally I think it's ok but maybe we can ask the community somehow? Maybe a poll on mailing list or g+?

**Comment by arthurvr** [comment]
> Do we care about 0.5 results?

FWIW I personally don't think we should anymore.

**Comment by samccone** [comment]
> Do we care about 0.5 results?

I would argue it is pretty critical to keep old docs around... pretty much forever

based on analytics I have after doing this for Marionette

http://marionettejs.com/docs/current/
http://marionettejs.com/docs/v1.8.5/

http://marionettejs.com/annotated-src/v2.0.1/

We still get tons and tons of hits and people have reached out to say how much they appreciate them..

**Comment by robdodson** [comment]
@samccone to be clear, we're not talking about deleting the docs but removing them from Google Search. Users will still be able to access them via the dropdown menu. Looking at analytics we can see that even though our site already has a search feature, most folks don't use it, so I don't feel like having it scoped to different doc sets will help that much.

I would wager that the majority of our doc searching comes from Google Search and if folks keep ending up in 0.5 docs by default that's a bummer. We can also hand map the url redirects when you click the dropdown but that doesn't fix the Google Search problem, which is the primary one I think we should target.

**Comment by ebidel** [comment]
0.5 docs will definitely still be available to browse on the site. This is about making 1.0 (and future content) more findable. We'll continue to port articles and other great 0.5 content.

LGTM

**Comment by samccone** [comment]
:+1: kk

**Comment by addyosmani** [comment]
LGTM to mothball. I would do this sooner rather than later so we have time to clean up any remaining weeds we've missed before the summit.

**Pull request merged**